backend/experiment/forms.py

```diff
@@ -145,6 +145,9 @@ class Meta:
         model = Experiment
         fields = ['name', 'slug', 'active', 'rules',
                   'rounds', 'bonus_points', 'playlists', 'experiment_series']
+        help_texts = {'consent': 'Enter the text to ask consent for using the experiment data for this instance of the experiment.<br> \
+                      This field will override any text loaded from the rules file.<br> \
+                      Allowed tags: HTML (default), MARKDOWN (Optional setting in the rules file)'}
```

Comment:
Do you mean "allowed formats" here? I find this help text slightly confusing. What does "optional setting in the rules file" mean?